Libraries/D3D12RaytracingFallback/src/TreeletReorderBindings.h

```diff
@@ -28,22 +28,50 @@ struct InputConstants
 #define NumTrianglesBufferRegister 1
 #define AABBBufferRegister 2
 #define ElementBufferRegister 3
+#define BaseTreeletsCountBufferRegister 4
+#define BaseTreeletsIndexBufferRegister 5
 
 #define GlobalDescriptorHeapRegister 0
```

Comment:
Good catch on the removal here, you can also remove these GlobalDescriptorHeap defines also